frontend/src/components/auth/Signup/AccountDetails.tsx

```diff
@@ -98,8 +92,6 @@ const AccountDetails = ({
       onOpen();
       return false;
     }
-    await setAuthenticatedUser(user);
```

Comment:
I removed this because otherwise the user was being signed in even when they didn't verify their email. Thoughts? Current implemented flow is they'll need to login after this verification.